### DISCUSSION TITLE
Multi Taxonomy property filter not working in Query Template as soon as more than one term is used

### DISCUSSION THREAD

**Discussion by Erwin-Git-Hub** [microsoft-search/pnp-modern-search#2647] [Q&A]
Version 4.8.0

On our search page we have a custom page property 'Domains' of type Managed Metadata.
In the Query Template of the PnP Search Results web part that we placed onto the search page we try to use the following:

IsDocument=1 {|tags:0{Page.Domains.termid}} 

This works fine as long as we only select 1 term, when selecting 2 or more terms, no results are returned anymore. 
Is this normal behavior, bad config/syntax or buggy?

Thanks

**Comment by wobba** [comment]
@Erwin-Git-Hub 
Tried just now and seems to expand correct.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/51104/a7b6f21f-6392-4e10-a362-3256ad2b7c26)
